cli/src/main/kotlin/utils/ConfigDirHelper.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.cli.utils
+
+import java.io.File
+
+import org.eclipse.apoapsis.ortserver.cli.COMMAND_NAME
+
+import org.ossreviewtoolkit.utils.common.Os
+import org.ossreviewtoolkit.utils.common.safeMkdirs
+
+/**
+ * The directory where the configuration files are stored, respecting the XDG Base Directory specification [1].
+ *
+ * [1]: https://specifications.freedesktop.org/basedir-spec/latest/
+ */
+internal val configDir: File
+    get() {
+        val dir = when {
+            Os.isLinux -> File(
+                System.getenv("XDG_CONFIG_HOME") ?: "${Os.fixupUserHomeProperty()}/.config/$COMMAND_NAME"
```

Comment:
Even if this case is for Linux, you should generally prefer `Os.env[]` to look up environment variables, as it does the right thing on Windows and looks them up case-insensitively. Similar below.